models/hr/src/index.ts

```diff
@@ -328,9 +328,9 @@ export function createModel (builder: Builder): void {
   createAction(
     builder,
     {
-      action: view.actionImpl.Delete,
-      label: view.string.Delete,
-      icon: view.icon.Delete,
+      action: view.actionImpl.Archive,
```

Comment:
Why not add action to core.class.Space on one place?

Reply:
Archive is reversible operation, so why not? The question is what to do with Employees being part of archived department.

Reply:
@haiodo unset their department?